st2client/st2client/commands/action.py

```diff
@@ -1343,19 +1343,28 @@ def __init__(self, resource, *args, **kwargs):
 
     @add_auth_token_to_kwargs_from_cli
     def run(self, args, **kwargs):
-        return self.manager.pause(args.id)
+        responses = []
+        for execution_id in args.id:
```

Comment:
Can we change to `args.ids` while we are at it? And change the description of the args to indicate it can be a list.